proteus/Domain.py

```diff
@@ -692,67 +723,56 @@ def __init__(self,fileprefix):
         D_base.__init__(self,3,name=fileprefix)
         self.meshfile=fileprefix
 
+
 class PiecewiseLinearComplexDomain(D_base):
     """
     3D domains desribed by closed surfaces made up of general polygonal facets.
     """
-    def __init__(self,fileprefix=None,vertices=None,facets=None,
-                 facetHoles=None,holes=None,regions=None,vertexFlags=None,facetFlags=None,regionFlags=None,regionConstraints=None,
-                 name="PLCDomain",units="m"):
+    def __init__(self, fileprefix=None, vertices=None, segments=None, facets=None,
```

Comment:
It is actually not used in any functions of `PiecewiseLinearComplexDomain`, so it is not necessary to pass them, it does not break anything to have them there but should it be removed?